free-1.md

```diff
@@ -10,41 +10,157 @@ _What need is met by your idea? Who are you helping? What is that person's
+since everything should be mathematical symbols.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would read the entire file to calculate how much branching is
+requested. 
+It would also calculate how long each statement in the tree is based on how
+many symbols are used.
+This is then used to calculate the angle and length of lines in the tree for
+branching to give each statement enough space.
+
+If the program runs with no issues, it would then produces a new file, an image 
+or pdf, of the resulting tree.
+Else, it would produce a file containing error messages.
+
+Possible errors could include unrecognized commands for symbols.
+With the syntax described before, blank lines could be an error or a warning.
+An error could arise if the user tries to branch in more than two directions,
+this would throw off the distance calculation.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to draw binary trees with logic symbols.
+
+It should be possible, but maybe not necessarily easy, to customize these
+symbols. 
+
+It should be impossible to draw trees that are not binary.
+Drawing shapes other than trees is also possible after a lot of difficulty.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are many packages in LaTeX for drawing trees.
+This is the code I actually used to generate the tree image used as an example
+earlier
+```
+\Tree
+[.{
+$(A\vee B) \wedge C\checkmark{}$    \\
+$C$ \\
+$A\vee B\checkmark{}$   \\
+}
+    [.{$A$}
+        [.{branch1$\quad$}
+        ]
+        [.{branch2}
+        ]
+    ]
+    [.{$B$}
+    ]
+]
+```
+I used the package qtree, which is actually for linguists so it uses
+linguistics conventions. The code shows that it requires a lot of work-around
+to get the tree I wanted. I used qtree with the help of 
+[this page](https://github.com/cpence/latex-tableaux)
+which documents ways to make qtree work for logic proofs.
+
+As mentioned earlier, tree packages generally expect each node to be a single
+line.
+So stacking in tableau trees do not translate neatly into most tree packages.
+
+There is a package for tableau trees, 
+[prooftrees](http://ctan.org/pkg/prooftrees).
+But the syntax is quite complicated since it offers a lot of customization,
+which is great for experienced users. But this makes it difficult for someone
+who wants to draw a basic tableau tree.
```

Comment:
As mentioned before, it would be cool if the language could also spit out LaTeX code for various backends as well as images/pdfs.